src/CommunityToolkit.Maui/Behaviors/AnimationBehavior.shared.cs

```diff
@@ -115,7 +115,7 @@ async Task OnAnimate(CancellationToken token)
 		}
 		catch (Exception ex) when (Options.ShouldSuppressExceptionsInAnimations)
 		{
-			Trace.WriteLine(ex);
+			Trace.TraceError(ex.StackTrace);
```

Comment:
I think we should rely on the full informartion

```suggestion
			Trace.TraceError(ex);
```